pkg/routeagent_driver/handlers/mtu/mtuhandler.go

```diff
@@ -19,30 +19,30 @@ limitations under the License.
 package mtu
 
 import (
+	"strconv"
 	"strings"
 
 	"github.com/pkg/errors"
 	submV1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
 	"github.com/submariner-io/submariner/pkg/event"
-	"github.com/submariner-io/submariner/pkg/ipset"
 	"github.com/submariner-io/submariner/pkg/iptables"
+	netlinkAPI "github.com/submariner-io/submariner/pkg/netlink"
 	"github.com/submariner-io/submariner/pkg/routeagent_driver/constants"
 	"k8s.io/klog"
-	utilexec "k8s.io/utils/exec"
+)
+
+const (
+	maxOverlayHeaderSize = 120
```

Comment:
Please add some notes on why/how we derived at the value of 120 for future reference.